vcr/config.go

```diff
@@ -30,6 +30,8 @@ type Config struct {
 	OIDC4VCI OIDC4VCIConfig `koanf:"oidc4vci"`
 	// datadir holds the location the VCR files are stored
 	datadir string
+	// strictmode holds a copy of the core.ServerConfig.Strictmode value
```

Comment:
I find this an ugly approach, why not just add it as property to `vcr`?

Reply:
also moved datadir.